src/routes/user.tsx

```diff
@@ -381,7 +425,13 @@ const UserProfileCard = ({
           />
         </dl>
         {editable && !isCurrentUser && <DeleteUserButton user={user} />}
-        {editable && <SetPasswordButton user={user} />}
+        {editable && (
+          <SetPasswordButton
+            user={user}
+            self={isCurrentUser}
```

Comment:
@varCepheid Do you want to rename the prop to `isCurrentUser`? I think that name is more clear than `self`, which makes me think of [other things](https://en.wikipedia.org/wiki/This_(computer_programming))